convert2rhel/main.py

```diff
@@ -163,6 +163,11 @@ def pre_ponr_conversion():
     # package analysis
     loggerinst.task("Convert: List third-party packages")
     pkghandler.list_third_party_pkgs()
+
+    # handle special cases
+    loggerinst.task("Convert: check and resolve possible special cases.")
+    special_cases.check_and_resolve()
```

Comment:
```suggestion
    loggerinst.task("Convert: resolve possible edge cases.")
    special_cases.check_and_resolve()
```

Reply:
@oamg/convert2rhel how do you think we should call this special cases python module. I am ok with the idea of @SpyTec . I'd like to have one name:
- for longer task (Convert: resolve possible {name}.)
- for python module

Reply:
What about `checks.py`? We'll be adding more checks in close future (e.g. for HA clustering, or for making sure repos passed through `--enablerepo` are accessible).

Reply:
Not really a check if it makes changes to the system though

Reply:
You're right. Then I suggest `workload.py` or `apps.py`. Under that we could have:
- this java check
- checks requested by stakeholders:
  - pacemaker checks
  - HA cluster check
  - check for used kmods N/A in RHEL
- check for packages N/A in RHEL
- possibly move there the existing check for third party pkgs (`list_third_party_pkgs`)

Reply:
I think I understand your idea @bocekm. you were right in saying that the heaviest work in the development is naming:)  However, I would split checks and workarounds.

workload - seams not intuitive name to me
apps - assumes some apps registration and discovery mechanism... 

Maybe let's create a separate ticket to refactor checks and special cases, but for this MR leave it more or less as it is now. I am ok to change special_cases to edge_cases or checks.py.

Reply:
What about

- `mischandler`,
- `systemhandler`, or just
- `system`

Reply:
I'll create a Jira ticket for that. Let's resolve this comment for now.

Reply:
The ticket is OAMG-4474

Reply:
I'd still change the task string to "Convert: resolve possible edge cases"

Reply:
@SpyTec will do